src/BaGet.UI/src/index.tsx

```diff
@@ -1,13 +1,14 @@
 import { initializeIcons } from '@uifabric/icons';
 import 'bootstrap/dist/css/bootstrap.css';
 import { BrowserRouter as Router, Route } from 'react-router-dom';
-
+import { config } from './config';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import App from './App';
 
 import DisplayPackage from './DisplayPackage/DisplayPackage';
 import Upload from './Upload';
+import createBrowserHistory from 'history/createBrowserHistory';
```

Comment:
What's this for?

Reply:
Forgot to remove it.
I tried adding the history to react router and setting the base path via the history.
But dom router does not support history so I had to use the basename option.